src/Components/LicenseDetails/LicenseDetails.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {Descriptions} from "antd";
+import {LicenseInfo} from "reduct-js";
+import "./LicenseDetails.css";
+import LicenseAlert from "../LicenseAlert/LicenseAlert";
+// @ts-ignore
+import prettierBytes from "prettier-bytes";
+
+interface LicenseDetailsProps {
+  license: LicenseInfo;
+  usage: bigint;
+}
+
+const LicenseDetails: React.FC<LicenseDetailsProps> = ({license, usage}) => {
+  const expiryDate = new Date(license.expiryDate).toLocaleDateString(undefined,
+    {
+      year: "numeric",
+      month: "long",
+      day: "numeric",
+    }
+  );
+
+  const hasExpired = Date.now() > license.expiryDate;
+  const isOverQuota = usage > BigInt(license.diskQuota);
+  const alertExplanation = hasExpired ? "Your license has expired. " : isOverQuota ? "You have exceeded your disk quota. " : "";
+
+  return (
+    <>
+      {alertExplanation && <div className="licenseAlert"><LicenseAlert alertExplanation={alertExplanation} /></div>}
```

Comment:
![image](https://github.com/reductstore/web-console/assets/67068/cd94d530-4796-4fbf-a411-243130c5f487)

I'm not sure if the mentioning of BUSL is relevant here. Maybe we could say that they must contact us to extend it.